src/test/java/core/mate/academy/service/MachineServiceTest.java

```diff
@@ -1,7 +1,6 @@
 package core.mate.academy.service;
 
 import static org.junit.Assert.assertEquals;
-
```

Comment:
needless change. static imports should be separated with empty line from non-static imports